### ISSUE TITLE
Private comments and replys

### ISSUE THREAD

**Issue by hoergen** [friendica/friendica#6335]
I would love to have a feature where you can comment a public or non public post with a private comment, that goes only to the post creator. And If the post creator replies the replay goes only to the private commenter.

Like
Publi post
 - comment
 - comment
 - private comment
 -- private reply
--- private reply reply
 - comment

**Comment by MrPetovan** [comment]
If I'm not mistaken, this can only works if the reply is sent to the comment you are replying to. However, at the moment, comments are sent to the OP no matter what. This ensures the reply author doesn't know the complete list of people the OP has shared his post with.

So, in order for this to work, we would have to adopt a radically different comment distribution approach, and I'm not sure @annando would agree with this change.

**Comment by hoergen** [comment]
I'm not sure if you understand what I mean. When I read a post and want to privately comment this post, so that the comment is just readable for the post creator, not for some other commenters. I don't think you have to change anything in the distributing model.

The Friendica node has to additionally choose if it distributes to all commenters or only to the private commenter.

Get me right. It's not that I choose who can read my comment. This is fixed to only between the creator and the commenter.

**Comment by annando** [comment]
Currently the post permissions are stored on the toplevel post. Also the toplevel post cares about distributing the post to the different receivers. This all had to be changed - which will be complicated.

**Comment by hoergen** [comment]
And if you just add a flag or something on a comment, that tells the server to handle it differently to a normal comment?

**Comment by hoergen** [comment]
> but it may be a confusing feature if we don't get the display right.

Yes the UI should display it very obviously maybe with a additional lock symbol (and)or different color.
